src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -746,8 +746,11 @@ static function (ChildDefinition $definition, AsPeriodicTask|AsCronTask $attribu
                 }
             );
         }
-        $container->registerAttributeForAutoconfiguration(JsonEncodable::class, static function (ChildDefinition $definition): void {
-            $definition->addTag('json_encoder.encodable');
+        $container->registerAttributeForAutoconfiguration(JsonEncodable::class, static function (ChildDefinition $definition, JsonEncodable $attribute): void {
+            $definition->addTag('json_encoder.encodable', [
+                'item' => $attribute->asItem,
```

Comment:
Why is it called `item` if the type is `Type::object`. I think `object`/`asObject` is more appropriate and match with [JSON name](https://www.w3schools.com/js/js_json_datatypes.asp).

Reply:
`list` is a better name than JSON `array` to remove any confusion with PHP array that can be a hashmap (object). And the method is already `Type::list`.